tests/integration/routes/test_feedback.py

```diff
@@ -62,20 +62,91 @@ def test_valid_feedback(self):
         self.assertInUrl(self.SENT_FEEDBACK_URL)
         self.assertInBody("Thank you for your feedback")
 
-    def test_second_submission(self):
-        # Given I launch and complete the test_feedback questionnaire, and provide feedback
+    def test_feedback_error_message_on_get_when_limit_reached(self):
+        # Given I launch and complete the test_feedback questionnaire, and provide
+        # feedback the maximum number of times
+        self._launch_and_complete_questionnaire()
+
+        for _ in range(0, EQ_FEEDBACK_LIMIT):
```

Comment:
Only minor but with the other limit tests (IR, email) we set a lower value for the limit in `SetUp` so wouldn't need these loops